components/dropdown/Dropdown.js

```diff
@@ -1,133 +1,92 @@
-import React, { useState, useEffect, useRef, useImperativeHandle } from 'react';
+import React, { useEffect, useRef } from 'react';
 import PropTypes from 'prop-types';
 import keycode from 'keycode';
-import { Icon } from 'react-components';
+import { classnames } from '../../helpers/component';
+import { usePopper, Popper } from '../Popper';
+import { noop } from '@babel/types';
 
-const ALIGN_CLASSES = {
-    right: 'dropDown--rightArrow',
-    left: 'dropDown--leftArrow'
-};
-
-const Dropdown = React.forwardRef(
-    (
-        {
-            isOpen,
-            content,
-            title,
-            children,
-            className,
-            autoClose,
-            autoCloseOutside,
-            align,
-            narrow,
-            loading,
-            disabled,
-            caret
-        },
-        ref
-    ) => {
-        const [open, setOpen] = useState(isOpen);
-        const wrapperRef = useRef(null);
-        const handleClick = () => setOpen(!open);
+const Dropdown = ({
+    anchorRef,
+    children,
+    originalPlacement = 'bottom',
+    close = noop,
```

Comment:
I'd prefer to call this `onClose` instead of `close` for consistency with other handlers, what do you think?

Reply:
yep, good point